src/Nethermind/Nethermind.Synchronization/SnapSync/ProgressTracker.cs

```diff
@@ -482,10 +501,50 @@ private void LogRequest(string reqType)
                 if (_logger.IsInfo)
                 {
                     string stateRangesReport = $"Snap         State Ranges (Phase 1): ({progress,8:P2}) {Progress.GetMeter(progress, 1)}";
-                    if (_lastStateRangesReport != stateRangesReport)
+                    if (progress >= 0.995)
```

Comment:
Maybe always have info about storage not only last 5%?
Can you show how current logs look with those changes?

Reply:
We don't know the total number of storage until all address range has been downloaded.